### ISSUE TITLE
Local client becomes unusable after first run

### ISSUE THREAD

**Issue by hdmi** [qdrant/qdrant-client#429]
I have created a local hybrid collection for dense and sparse vectors, and after populating it with some vector, the next time I try to reopen the client creation fails.

Here is a small proof of concept 

```python
def testCloseOpenDB():
    collectionName = "asdf"
    db = QdrantClient(path="asdf")
    db.recreate_collection(
        collection_name=collectionName,
        vectors_config=models.VectorParams(
            size=1024, 
            distance=models.Distance.COSINE),
        sparse_vectors_config={
                "text-sparse": models.SparseVectorParams(
                index=models.SparseIndexParams(on_disk=True)
        )},
    )

    # Insert some sparse vectors
    for i in range(0,10):
        operation_info = db.upsert(
            collection_name=collectionName,
            wait=True,
            points=[PointStruct(id = i, 
                                vector = { "text": models.SparseVector(indices=[0,100,123,3213], values=[0.12321, -1.3123, 0.9321, -1.33333])},
                                payload = {"name": f"Name_{i}"})
                    ],
            )
        print(operation_info)

    db.close()

testCloseOpenDB() # First run, creation of collection and insertion
testCloseOpenDB() # Second run, fails to create the client
```

And here is the error

```
Exception has occurred: TypeError
'VectorParams' object is not subscriptable
  File "/Users/asdf/projects/lse/scrap.py", line 388, in testCloseOpenDB
    db = QdrantClient(path="asdf")
         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asdf/projects/lse/scrap.py", line 415, in <module>
    testCloseOpenDB() # Second run, fails to create the client
    ^^^^^^^^^^^^^^^^^
TypeError: 'VectorParams' object is not subscriptable
```

Running on MacOS
qdrant-client 1.7.0
Python 3.11

**Comment by joein** [comment]
Hi @hdmi 

Thank you for pointing it out and for a code sample (which occurred to be really valuable :) )!

It seems that there is a bug on our side, which does not correctly load unnamed vectors

As a workaround until we fix it, I can suggest you to make dense vectors named:

```
vectors_config=models.VectorParams(
            size=1024, 
            distance=models.Distance.COSINE)
```
->
```
vectors_config={"text": models.VectorParams(
            size=1024, 
            distance=models.Distance.COSINE)
}
```

By the way, there is another bug in local mode, which is also highlighted by your code sample :(

```
operation_info = db.upsert(
            collection_name=collectionName,
            wait=True,
            points=[PointStruct(id = i, 
                                vector = { "text": models.SparseVector(indices=[0,100,123,3213], values=[0.12321, -1.3123, 0.9321, -1.33333])},
                                payload = {"name": f"Name_{i}"})
                    ],
            )
```

Here you are trying to upsert vector `text` instead of `text-sparse`. `text` vector does not exist and should throw an error

However with correct configuration it should work as intended.

We will fix these bugs in the upcoming days, sorry for the inconvenience

**Comment by hdmi** [comment]
Ok, thanks for the insights!

Kind of offtopic but related to the bug, is there a way of recovering the already stored embeddings?
I have days of generated sparse and dense vectors that I would like to recover.

Maybe I can build a qdrant server and place the sqlite files there.

Thank you for the support!

**Comment by joein** [comment]
Yes, there are several ways to access your points

1) As it is persisted in sqlite, you can just read the data from sqlite using `sqlite3`

Something like this:

```python
import sqlite3
import pickle
connection = sqlite3.connect('asdf/collection/asdf/storage.sqlite')
cursor = connection.cursor()
cursor.execute("SELECT point FROM points")
for row in cursor.fetchall():
     rows = pickle.loads(row[0])

print(rows)
```

2) The same thing but using QdrantClient classes
```python
from qdrant_client.local.persistence import CollectionPersistence

col = CollectionPersistence("asdf/collection/asdf")
for point in col.load():
    print(point)
```

3) You can modify `load_vectors` method in `qdrant_client.local.local_collection`

```python
vectors[name].append(
    np.ones(self.config.vectors[name].size, dtype=np.float32)
)
```

->

```
size = self.config.vectors[name].size if isinstance(self.config.vectors, dict) else self.config.vectors.size
vectors[name].append(np.ones(size, dtype=np.float32))
```

4) You can wait a little bit for the next release where we will fix it

**Comment by joein** [comment]
available as of `qdrant-client==1.7.1`
